packages/frontend/@n8n/i18n/src/locales/en.json

```
@@ -2559,6 +2559,7 @@
 	"workflowSettings.showMessage.saveSettings.title": "Workflow settings saved",
 	"workflowSettings.timeoutAfter": "Timeout After",
 	"workflowSettings.timeoutWorkflow": "Timeout Workflow",
+	"workflowSettings.executionTimeout": "Timeout Workflow",
```

Does this need to match the others? And should it be "Timeout Execution"?

```suggestion
	"workflowSettings.timeoutExecution": "Timeout Execution",
```